README.md

```diff
@@ -38,6 +39,15 @@ module "power-infrastructure" {
 }
 ```
 
+You can configure the details in a `input.tfvars` variable definitions file.
+
+```bash
+terraform apply -var-file="input.tfvars"
+```
+
+## Required IAM access policies
+
+No permissions are needed to run this module.
```

Comment:
Someone will need to find out required permissions for this section. But permissions are required, so perhaps put a TBD for now and create GIT issue to add them?

Reply:
@surajsbharadwaj Do you have these handy? For example, here is what we have for other modules -> https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone-vsi#required-iam-access-policies

Reply:
@ocofaigh , no idea. @kbiegert knows may be. He added few stuff to catalog.

Reply:
This is nothing to do with catalog. This is IAM permissions. We need to document what exact IAM permissions are required to provision all of the resources in IBM cloud

Reply:
I have updated the text to say "TBC" and created an internal issue to revisit and fill the permissions in so that we can proceed and merge this PR